opta/commands/config.py

```diff
@@ -0,0 +1,54 @@
+import click
+from click_didyoumean import DYMGroup
+
+from opta.core.aws import AWS
+from opta.core.cloud_client import CloudClient
+from opta.core.gcp import GCP
+from opta.exceptions import UserErrors
+from opta.utils import logger
+
+
+@click.group(cls=DYMGroup)
+def config() -> None:
+    """
+    Manage opta configurations
+    """
+    pass
+
+
+cloud_option = click.option(
+    "--cloud",
+    help="The cloud provider to use",
+    required=True,
+    type=click.Choice(["aws", "google", "azurerm"]),
+)
+
+
+@config.command()
+@cloud_option
+def view(cloud: str) -> None:
+    """
+    View the opta configuration file for a given cloud provider
+
+    Use the current cloud credentials to fetch the remote opta config file
+    """
+    if cloud.lower() == "azurerm":
+        raise UserErrors("Currently AzureRM isn't supported.")
+    cloud_client: CloudClient = __get_cloud_client(cloud)
+    detailed_config_map = cloud_client.get_all_remote_configs()
+    if detailed_config_map:
+        for bucket, detailed_configs in detailed_config_map.items():
+            for config_name, actual_config in detailed_configs.items():
+                logger.info(
+                    f"# Bucket Name: {bucket}\n# Config Name: {config_name}\n{actual_config['original_spec']}\n"
+                )
+
+
+def __get_cloud_client(cloud: str) -> CloudClient:
+    cloud_client: CloudClient
+    if cloud.lower() == "aws":
+        cloud_client = AWS()
+    elif cloud.lower() == "google":
+        cloud_client = GCP()
+
+    return cloud_client
```

Comment:
JD needed something similar, we can consolidate such methods in the same place
https://github.com/run-x/opta/blob/3960cd526ce82299dfeab42d366d7b2da48cbd7d/opta/layer.py#L176

Reply:
+1

Reply:
Sure.

Reply:
For this one, we can have another PR and figure out where can we put this method.